docs/installation.md

```diff
@@ -85,15 +88,14 @@ Then solaar can be run as /usr/local/bin/solaar.
 [pip]: https://en.wikipedia.org/wiki/Pip_(package_manager)
 
 
-
-### Running Solaar at Startup
+## Running Solaar at Startup
 
 Solaar is run automatically at user login using a desktop file,
 which may have been installed at `/etc/xdg/autostart/solaar.desktop`.
 If you manually install Solaar you may need to modify this automatic starting of Solaar.
```

Comment:
“may need to modify” is here meant that the user should copy this file them selfs? Or should the user create such file himself at all. I wondering which are meant, and if it is not more clear to just mention them.

Reply:
Yeah that should be fixed up.  One problem is that installing Solaar yourself (even globally) may result in an autostart file that is not working because it starts a version in /usr/bin (instead of /usr/local/bin) or even to a file that does not exist.

I change the sentence to 
```
If you install Solaar manually you may need to create or modify this file.
```